commons/src/main/scala/io/udash/web/commons/views/MarkdownView.scala

```diff
@@ -0,0 +1,42 @@
+package io.udash.web.commons.views
+
+import com.avsystem.commons._
+import com.avsystem.commons.misc.AbstractCase
+import io.udash._
+import io.udash.web.guide.markdown.{MarkdownPage, MarkdownPageRPC}
+import io.udash.web.guide.styles.MarkdownStyles
+
+import scala.util.{Failure, Success}
+
+trait MarkdownPageState extends State {
+  def page: MarkdownPage
+}
+
+final case class MarkdownPageViewFactory[T <: MarkdownPageState]()(
+  rpc: MarkdownPageRPC
+) extends AbstractCase with ViewFactory[T] {
+  override def create(): (MarkdownView, MarkdownPresenter[T]) = {
+    val content: Property[String] = Property("")
+    (new MarkdownView(content), new MarkdownPresenter[T](content, rpc))
+  }
+}
+
+final class MarkdownPresenter[T <: MarkdownPageState](
+  content: Property[String],
+  rpc: MarkdownPageRPC
+) extends Presenter[T] {
+  override def handleState(state: T): Unit = {
+    rpc.loadContent(state.page).onCompleteNow {
+      case Success(rawHtml) => content.set(rawHtml)
+      case Failure(exception) => content.set(exception.toString)
```

Comment:
After deleting the `i18n.md` file and trying to see the view, after 30 second I got a poorly formatted `io.udash.rpc.internals.UsesServerRPC$CallTimeout: Response missing after 30 seconds.` message. The Future didn't complete for some reason. :( 

I'd much rather see a nicely formatted `File does not exist` message in that case.

Reply:
Seems to work for me, can you commit some nice formatting of your liking? :pray:

Reply:
`CallTimeout` was my mistake, sorry :( I've styled exceptions a bit nicer, take a look.